src/GitVersionTfsTask/Tasks/GitVersion/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "git-version",
+  "version": "1.0.0",
+  "private": true,
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "vsts-task-lib": "0.9.20",
+    "q" : "1.4.1"
```

Comment:
vsts-task-lib depends on q. I guess this is related to the fact that some agents are running older versions of node which won't provide native Promises.